packages/seacas/libraries/ioss/src/Ioss_Utils.C

```diff
@@ -580,15 +580,15 @@ namespace {
         // It is possible that the first name(s) that match with two
         // suffices have a basename that match other names with only a
         // single suffix lc_cam_x, lc_cam_y, lc_sfarea.
-        for (int i = index + 1; i < num_names; i++) {
-          char                    *tst_name = names[i];
+        for (int id = index + 1; id < num_names; id++) {
```

Comment:
Good catch.  I thought I had fixed this, but definitely didn't..  Maybe I accidentally fixed on a branch...